content/en/docs/instrumentation/js/serverless.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Serverless
+weight: 8
+---
+
+This guide will show you how to get started with tracing serverless functions using OpenTelemetry instrumentation libraries.
+
+## AWS Lambda
+
+Following the next steps will allow your AWS Lambda functions to use OpenTelemetry to send traces to the configured backend.
+
+### Dependencies
+
+Create an empty package.json:
+
+```sh
+npm init -y
+```
+
+Install dependencies used by the example.
+
+```sh
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/auto-instrumentations-node \
+  @opentelemetry/exporter-trace-otlp-http \
+  @opentelemetry/instrumentation \
+  @opentelemetry/sdk-trace-base \
+  @opentelemetry/sdk-trace-node
```

Comment:
I think you don't need all of them because they have some interdependencies, can you verify what the smallest set of packages is, that you really need?

Reply:
Just verified that this is the smallest number of packages that I need